BookingsApi/BookingsApi.Contract/Requests/Enums/UpdateBookingStatus.cs

```diff
@@ -5,8 +5,9 @@
     /// </summary>
     public enum UpdateBookingStatus
     {
-        Created = 1,
-        Cancelled = 2,
-        Failed = 3
+        Booked = 1,
```

Comment:
For consistency with `BookingStatus` it would be good to see the comments added here for swagger